modules/consul-cluster/variables.tf

```diff
@@ -164,6 +164,17 @@ variable "root_volume_delete_on_termination" {
   default     = true
 }
 
+variable "root_volume_encrypted" {
+  description = "Encrypt the volume at rest"
+  default     = false
+}
+
+variable "ebs_block_devices" {
+  description = "List of EBS volumes"
+  type        = list
```

Comment:
This should be a map, not a list so that (a) the `for_each` works and (b) each volume has a unique name/ID, so if a user removes one or adds one, it only affects that one, rather than all the items after it in a list.